nethsm-api.yaml

```diff
@@ -787,15 +787,9 @@ paths:
         "200":
           description: ""
           content:
-            application/x-pem-file:
-              schema:
-                $ref: "#/components/schemas/PemCert"
-            application/x-x509-ca-cert:
+            application/octet-stream:
               schema:
                 type: string
```

Comment:
Having `application/octet-stream` with a `string` type is confusing for me.  Does it use binary or base64?  Either the schema should be omitted to indicate binary, or the base64 encoding should be set explicitly.

Reply:
Looks like it’s just raw binary.  In this case it would be clearer to just omit the `schema`.  Alternatively, you could set `format: binary`, cf. https://github.com/Nitrokey/nethsm-sdk-py/blob/ea144f4f14a96fc8ca5013e16f96496d699f5452/nethsm-api.yaml#L1723